runtime/bajun/src/lib.rs

```diff
@@ -588,6 +595,9 @@ construct_runtime!(
 		Treasury: pallet_treasury = 41,
 		Council: pallet_collective::<Instance2> = 42,
 		CouncilMembership: pallet_membership::<Instance2> = 43,
+
+		// Indexes 50-59 should be reserved for our games.
+		AAA: pallet_ajuna_awesome_avatars = 50,
```

Comment:
Let's rename (otherwise Polkadot UI displays it as `aaa` which could be confusing to users):
```suggestion
		AwesomeAvatars: pallet_ajuna_awesome_avatars = 50,
```

Could you also rename in solochain runtime?

Reply:
Done